explorer/src/lib/services/gql-queries.js

```diff
@@ -86,6 +85,12 @@ export const getLatestChainQueryInfo = (amount) => ({
   variables: { amount },
 });
 
+/** @param {string} id */
+export const getMempoolTx = (id) => ({
+  query: "query($id: String!) { mempoolTx(hash: $id) { json } }",
```

Comment:
I wouldn't retrieve the whole `json` field as we don't use it: just pick the smallest one.

Reply:
We probably will want to use the json in the next iteration, when we show the partial data?  Shouldn't we indicate the intention and keep this as it is?

Reply:
As it's just a field name I would just pick the smallest one (maybe `isDeploy`?) now.
Then we'll think about it when we do the next step.